tool/teleport/common/teleport.go

```diff
@@ -653,6 +653,21 @@ func onJoinOpenSSH(clf config.CommandLineFlags) error {
 		log.SetLevel(log.DebugLevel)
 	}
 
+	//Proxy Server and Token are required configuration so confirming they exist before continuing
```

Comment:
Extreme nit, only because I had a linter yell at me for this recently.
```suggestion
	// Proxy Server and Token are required configuration so confirming they exist before continuing
```